jetty-http/src/main/java/org/eclipse/jetty/http/HttpParser.java

```diff
@@ -1982,19 +1975,12 @@ private IllegalCharacterException(State state,HttpTokens.Token token,ByteBuffer
         }
     }
 
-    public interface ComplianceViolation
-    {
-        String getName();
-        String getDescription();
-        String getUrl();
-    }
-
     /**
      * Section Declarations, used for reporting Violations
      */
-    enum ViolationSection implements ComplianceViolation
+    enum RFC7230SpecReference implements SpecReference
     {
-        CASE_INSENSITIVE_FIELD_VALUE_CACHE("", "Use case insensitive field value cache"),
+        CASE_INSENSITIVE_FIELD_VALUE_CACHE("", "Use case insensitive field value cache"), // TODO: move out to HttpConfiguration proper
```

Comment:
This could also refer to https://tools.ietf.org/html/rfc7230#section-3.2, because by omission it implies that field values are case sensitive.  The key example is https://tools.ietf.org/html/rfc7230#section-6.1  where we might correct  `Connection: Close` to `Connection: close` with a case insensitive cache.

We probably need a longer description on this one saying exactly what it does.